### DISCUSSION TITLE
How do I center my QuickFolder tabs within the toolbar?

### DISCUSSION THREAD

**Discussion by amfonda** [RealRaven2000/QuickFolders#564] [Q&A]
I recently upgraded to TB 115 and I'm using QF 6.3.  In my previous version I had my QuickFolder tabs centered in the toolbar space.  I don't recall how I did that since there is no option for this in the current QF options.  If I'm missing something please advise.  If its not an option perhaps it should be.  Thanks.

**Comment by RealRaven2000** [comment]
Its definitely possible, you would need to use userChrome.css for that. I can’t tell you the detail right now because i am on annual leave and away from my computers for a few days but i can look at this again coming Sunday. If you don’t hear from me by next week, ping me here again to remind me.

**Comment by RealRaven2000** [comment]
https://quickfolders.org/faq_tips.html#UserChrome


![](https://quickfolders.org/images/Toolbar-Legend.png)

**Comment by amfonda** [comment]
I found this in my e-mail archives:

> Axel Grude wrote on 2/16/2018 7:42 AM:
>
> Dear Mark,
>
> yes you could create a userChrome.css in your Thunderbird profile/chrome folder and add the following rule:
> 
> #QuickFolders-FoldersBox {
>   text-align: center;
> }
> 
> This should do the trick.
> 
> to get to your profile open file explorer and enter %appdata% then find Thunderbird\Profiles\yourProfileName:
> 
> C:\Users\Axel\AppData\Roaming\Thunderbird\Profiles\aj24194hh.default\chrome
> 
> If there is no "chrome" subfolder, create one, then create the file userChrome.css and paste the three lines above; restart Thunderbird. Voila.

Since my QF settings have migrated I still have that modified userChrome.css file in my profile... but it did not work even when QuickFolders was working (see #429).  Oh. well.

**Comment by RealRaven2000** [comment]
This might not work anymore as the tabs are now flex items,but its a start:

```css
#QuickFolders-FoldersBox   {
  text-align: center !important;
} 
```

According to the flex model documentation the code below might be more appropriate:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container

```css
#QuickFolders-FoldersBox   {
  justify-content: center !important;
} 
```

Instead of `center`, you would use `flex-start` for left alignment and `flex-end` for right alignment ( unless you use a RTL locale like hebrew were this is reversed. )

**Comment by amfonda** [comment]
This works to center all the tabs in the toolbar area:

```css
#QuickFolders-FoldersBox   {
  justify-content: center !important;
} 
```

**Comment by RealRaven2000** [comment]
OK, I just had another person ask about this, and also saw that [my FAQ pages](https://quickfolders.org/faq_tips.html#UserChrome) were out of date. While testing, also asked myself whether it might make sense to have the first row at the bottom and add more on top, here is what I came up with:

```css
#QuickFolders-FoldersBox {
  flex-wrap: wrap-reverse;
}
```

Even though unusual, this may actually be advantageous because the more "Important" first tabs are closer to the thread list:

![Image](https://github.com/user-attachments/assets/f0f737f9-22af-4748-9346-81e2c0bd947b)
